controllers/child-controller.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="de-DE">
+<head>
+    <meta charset="UTF-8">
+    <title>Kinder</title>
+</head>
+    <body>
+        
+        <header>
+            <h1>Unsere Kleinen</h1>
+            <p>Hier findest du kannst ob unsere talenten</p>
+        </header>
```

Comment:
```suggestion
            <h1>Unsere Kleinen</h1>
            <p>Hier findest du kannst ob unsere talenten</p>
```

Header ist eine zentrale Komponente, die für jede Seite gleich ist. In deinem Fall ist es eher ein Heading. Dafür gibt es aber keine Komponente an sich. Also kannst du den Header erstmal rauslöschen den Inhalt aber nicht.